### DISCUSSION TITLE
How long does it take you to generate a picture?

### DISCUSSION THREAD

**Discussion by WilliWantsToKnow** [lllyasviel/Fooocus#3317] [Q&A]
Hello everybody,
I would appreciate your input here:
I suspect my PC isn´t generating images as fast as it could. For reference: To generate a hyperrealistic image on the setting quality, jpeg, and aspect ratio 1:1, it takes me consistently 6 minutes (it varies only by about 10 seconds).  I have read in discussions and seen on YouTube videos that it (should?) only takes a couple of seconds.

How long do you take?

And a follow-up question:
What could cause you to have long generation times? The PC components. I have a PC with a Ryzen 7 5700 X, 32 GB of RAM, and a Radeon 6700 XT GPU. I know that is not a high-level PC but still if it actually only takes seconds to generate an image and I take 6 minutes that is too much of a difference right? 
Could it be because I am using an AMD GPU?
Or could it be something in the settings? My images always get generated in lowvram mode. Even though I edited the run.bat file by adding the line:
--always-high-vram

Is there something else that could slow down my PC?

Thank you for helping!

**Comment by mashb1t** [comment]
5900X, RTX 3080 (10GB VRAM), 32GB RAM, ~1.9it/s, ~22-25s for 1 quality image.
6700 XT is most likely your bottleneck, even with 12GB VRAM it isn't as performant as Nvidia counterparts for this usecase.
